apps/server/src/modules/h5p-editor/temporary-file-storage/temporary-file-storage.spec.ts

```diff
@@ -188,65 +213,87 @@ describe('TemporaryFileStorage', () => {
 					stream.on('end', resolve);
 				});
 				expect(content).not.toBe(null);
-				expect(content.toString()).toEqual(testContent);
+				expect(content.toString()).toEqual(actualContent);
 			});
 		});
 		describe('WHEN file does not exist', () => {
 			it('should throw error', async () => {
-				const { user } = setup();
-				await expect(async () => service.getFileStream('abc/nonexistingfile.txt', user)).rejects.toThrow();
+				const { user1 } = setup();
+				repo.findByPath.mockImplementation(() => {
+					throw new Error('Not found');
+				});
+				await expect(async () => storage.getFileStream('abc/nonexistingfile.txt', user1)).rejects.toThrow();
 			});
 		});
 	});
 
 	describe('listFiles is called', () => {
 		describe('WHEN existing user is given', () => {
 			it('should return only users file', async () => {
-				const { user, filename } = setup();
-				const files = await service.listFiles(user);
+				const { user1, file1 } = setup();
+				repo.findByUser.mockResolvedValue([file1]);
+				const files = await storage.listFiles(user1);
 				expect(files.length).toBe(1);
-				expect(files[0].ownedByUserId).toBe(user.id);
-				expect(files[0].filename).toBe(filename);
+				expect(files[0].ownedByUserId).toBe(user1.id);
+				expect(files[0].filename).toBe(file1.filename);
 			});
 		});
 		describe('WHEN no user is given', () => {
 			it('should return both users files', async () => {
```

Comment:
It only returns the expired files. This should be denoted in the test.

Reply:
Renamed test to reflect reality